_sass/rootvariables.scss

```diff
@@ -1,88 +1,172 @@
-:root{
+:root {
+  // Colors that will not change in the dark mode.
+  --brand-color-primary: #00d3a9;
+  --brand-color-secondary: #00b39f;
+  --brand-color-secondary-50: rgba(0, 179, 159, 0.5);
+  --color-white: #fff;
+  --color-primary-light: #fff;
+  --color-primary-light-dark: #d9e0e2;
+  --color-primary-medium-dark: #ffffff;
+  --color-primary-medium: #f8f9fa;
+  --color-primary-dark: #f1f3f5;
+  --color-primary-extra-dark: #b2b6bd;
+  --color-primary-super-dark: #818c9a;
+  --color-secondary-light: #495057;
+  --color-secondary-medium: #343a40;
+  --color-secondary-dark: #212529;
+  --color-grey-light: #868e96;
+  --color-primary-light-tbase: #ecf0f3;
+  --color-primary-qmeta: #777777;
+  --color-primary-qtext: rgb(0, 0, 0);
+  --color-primary-testimonial-wrapper: rgb(250, 250, 250);
+  --color-nav-fixed: black;
 
-    // Colors that will not change in the dark mode.
-    --brand-color-primary:#00D3A9;
-    --brand-color-secondary:#00B39F;
-    --brand-color-secondary-50: rgba(0, 179, 159, .5);
-    --color-white:#fff;
-    --color-primary-light:#fff;
-    --color-primary-light-dark:#d9e0e2;
```

Comment:
@shourya2002-geek please remove all immaterial changes. These lines don't need to change and only make it difficult to review.